sktime/forecasting/mstl.py

```diff
@@ -0,0 +1,114 @@
+# !/usr/bin/env python3 -u
```

Comment:
What is this shebang for? Can you please explain why is it required?

Reply:
Not sure, I think I must have copy/pasted it from the LTSF code. Removing it now